packages/catalog-process/src/services/catalogService.ts

```diff
@@ -445,12 +445,22 @@ export function catalogServiceBuilder(
         }
       }
 
+      const updatedMode = apiEServiceModeToEServiceMode(eserviceSeed.mode);
+
+      const checkedRiskAnalysis =
+        updatedMode === eserviceMode.deliver &&
+        eservice.data.mode === eserviceMode.receive
+          ? []
+          : eservice.data.riskAnalysis;
```

Comment:
Is it really needed to check if the previous mode was `RECEIVE`?
```suggestion
      const checkedRiskAnalysis =
        updatedMode === eserviceMode.deliver ? [] : eservice.data.riskAnalysis;
 ```